infra/nnfw/cmake/CfgOptionFlags.cmake

```diff
@@ -23,6 +23,7 @@ option(BUILD_ONERT_RUN "Build onert_run" ON)
 option(BUILD_ONERT_TRAIN "Build onert_train" ON)
 option(BUILD_TFLITE_LOADER "Build TensorFlow Lite loader" ON)
 option(BUILD_CIRCLE_LOADER "Build circle loader" ON)
+option(BUILD_TRAININFO_LOADER "Build circle traininfo loader" ON)
```

Comment:
How about using `circlr+` word instead of `traininfo` ? Because this pr is for loading circle+ file format. :)

Reply:
I intentionally choose the word `traininfo` instead of `circle+`.

Because `circle+` range is a bit ambiguous. 
Which does `circle+` range for?
   (1) both `circle format` and `train info`
   (2) only `train info`
 
I thought `circle+` means (1). 
But this loader mainly focuses on loading `train info` from metadata. 
So, That's why I choose `traininfo loader`.

Reply:
I thought circle+ contains all meta information included in circle+ file format.
This circle+ loader loads all meta information from circle+ file. And each function can select the meta data it is interested in.

Reply:
Oh, I see your points.

But still for me, the words "circle+ loader" look like it is responsible for loading both circle(subgraphs and tensors) and its metadata(train info and .. etc).

Therefore, How about the word `circletr_meta_loader` ? 
+ ) I also thought the word `circle+_meta_loader`, but I'm afraid that string "+" cause trouble later.